### DISCUSSION TITLE
Truncate not working

### DISCUSSION THREAD

**Discussion by davea38** [tailwindlabs/tailwindcss#12468] [Help] [answered]
I can't get truncate to work in [this playground](https://play.tailwindcss.com/FLnYXVYH2k)

Could some kind person tell me why my lorem ipsum text in the paragraph is not truncating please? I'm guessing it's something to do with flex...?

**Comment by wongjn** [comment, marked as answer]
Indeed!

We can see why truncate is not working in devtools:
![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/174c04a9-f004-482c-b61c-daef3b723de3)

The `<p>` element is as wide as the text. Truncation only occurs when the text **overflows** the element, but here, the element increases its width to accommodate the text.

To rework this, we'd need to apply `min-width: 0` to some elements within the horizontal flex layouts to override the implicit `min-width: min-content`: https://play.tailwindcss.com/bk0oyVyWIK

![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/0366f5bf-8b0a-464b-adc9-cae27df419e2)

**Reply by davea38** [reply]
Brilliant, thank you for the quick response and explaining so well! That's been a head scratcher for the last hour :)
